doc/user-guides/dnspolicy/load-balanced-dns.md

```diff
@@ -9,7 +9,7 @@ This document will show you how to setup a load balanced DNS configuration using
 
 It is most useful to use the load balancing options when targeting multiple gateways that share a listener host E.G (api.example.com). It is also perfectly valid to use it when you only have a single gateway; this provides the benefit of allowing you to easily expand beyond this single gateway for a given shared hostname. It is worth knowing that the load balanced DNSpolicy comes with a relatively small additional cost of some added records and lookups during DNS resolution vs a "simple" DNSPolicy with no load balancing specified as the latter only sets up a simple A or CNAME record. So in summary if you expect to need multiple gateways for a given listener host then you should take advantage of the load balanced option.
 
-
+moving-between-simple-loadbalanced
```

Comment:
Is this intentional?

Reply:
nope... sigh doing too many things at once.